lib/browser/rpc-server.js

```diff
@@ -476,10 +481,22 @@ ipcMain.on('ELECTRON_BROWSER_SANDBOX_LOAD', function (event) {
   }
 })
 
-ipcMain.on('ELECTRON_BROWSER_GET_LAST_WEB_PREFERENCES', function (event) {
+const handleReturnValue = function (event, code) {
   try {
-    event.returnValue = [null, event.sender.getLastWebPreferences()]
+    event.returnValue = [null, code()]
```

Comment:
I don't really like a function named `code` =/

Reply:
me neither, what would you suggest instead?